### ISSUE TITLE
Allow infinite zooming out

### ISSUE THREAD

**Issue by philipwilk** [flxzt/rnote#917]
**Please submit only one feature request in one issue!**  
When I'm doing linear algebra, the document can get quite wide or tall, I usually zoom out really far so I dont have to scroll as much to navigate, but even this is impractical when the page gets really big and i have to scroll multiple screen widths or heights to get to my content.

**Is your feature request related to a problem? Please describe.**  
Navigating large documents is slow.

**Describe the solution you'd like**  
Allow zooming out practically infinitely.

**Describe alternatives you've considered**  
There isn't really any. Even if you could scroll faster, it wouldn't help much if you cant see where stuff is.

**Additional context**  
Realistically this is a rare edge case so isn't all that important.

**Comment by SomeonetookSomeonetookmeep** [comment]
Yeah, I also picked this up mostly do deal with my Calculus notes and problems. This goes very important for me, and hand in hand with infinite zoom much easier to move little details with things, though I definetly see zoom in as a seperate bit.

**Comment by amitnovick** [comment]
Having just infinite canvas without infinite zoom (and without bookmarks https://github.com/flxzt/rnote/issues/294) means you can't navigate a big canvas efficiently. +1 from me ➕
